### ISSUE TITLE
Sensor breaks with pyasn1 package

### ISSUE THREAD

**Issue by elgris** [instana/python-sensor#23]
Basically, you can find the suspect here:
```python
meter.py

...
    def collect_modules(self):
        try:
            m = sys.modules
            r = {}
            for k in m:
                if m[k]:
                    d = m[k].__dict__
                    if "version" in d and d["version"]:
                        r[k] = self.jsonable(d["version"])
                    elif "__version__" in d and d["__version__"]:
                        r[k] = self.jsonable(d["__version__"])
                    else:
                        r[k] = "builtin"

            return r
        except Exception as e:
            l.error("collect_modules: ", str(e))

            return None
...
```

Following line of code fails if module `pyasn1` is being processed:
```python
if "version" in d and d["version"]
```

exception thrown is `pyasn1.error.PyAsn1Error: Uninitialized ASN.1 value ("__nonzero__" attribute looked up)`

This problem came from package `pysnmp` which imports `pyasn1`

## Snippet to reproduce

You can reproduce this problem with a small snippet

```
import sys
from pysnmp.entity.rfc3413.oneliner import cmdgen

m = sys.modules
r = {}
for k in m:
    if m[k]:
        d = m[k].__dict__
        if "version" in d and d["version"]:
            r[k] = d["version"]
        elif "__version__" in d and d["__version__"]:
            r[k] = d["__version__"]
        else:
            r[k] = "builtin"
print(r)
```

**Comment by elgris** [comment]
Just in case: `pyasn1-0.3.5` was used

**Comment by pglombardo** [comment]
Thanks @elgris - this was very helpful.  Fixed in #24
